_data/meltano/extractors/tap-db2/danielptv.yml

```
@@ -0,0 +1,72 @@
+capabilities:
+- about
+- catalog
+- discover
+- state
+- stream-maps
+description: "Tap-DB2 is a Singer tap for IBM DB2 data sources."
```

```suggestion
description: IBM DB2 Relational Database
```

What do you think about making this consistent with the other variants so it doesnt change when toggling variant pages?